ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -179,239 +180,256 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{nam
-                                                                </div>
-                                                            </div>
-                                                            <div className='white-box__details'>
-                                                                <div className='row white-box__details-row'>
-                                                                    <div className='columns small-3'>Tags:</div>
-                                                                    <div className='columns small-9'>
-                                                                        {((metadata.tags || []).length > 0 && metadata.tags.join(', ')) || 'No tags'}
                                                                     </div>
-                                                                </div>
-                                                            </div>
-                                                            <div className='white-box__details'>
-                                                                <div className='row white-box__details-row'>
-                                                                    <div className='columns small-3'>Author:</div>
-                                                                    <div className='columns small-9'>{metadata.author}</div>
-                                                                </div>
-                                                            </div>
-                                                            <div className='white-box__details'>
-                                                                <div className='row white-box__details-row'>
-                                                                    <div className='columns small-3'>Message:</div>
-                                                                    <div className='columns small-9' style={{display: 'flex', alignItems: 'center'}}>
-                                                                        <div className='application-details__commit-message'>{renderCommitMessage(metadata.message)}</div>
+                                                                </React.Fragment>
+                                                            )
+                                                        }}>
+                                                        <div className='application-details__application-search-bar'>
+                                                            <DataLoader key='applications' load={() => AppUtils.handlePageVisibility(() => AppUtils.loadApplications())}>
+                                                                {(applications: models.Application[]) => (
+                                                                    <Query>{query => <SearchBar content={query.get('search')} apps={applications} ctx={ctx} />}</Query>
```

Comment:
Hi @ciiay, what do you think about this?  Try adding a new property `currentApp` on SearchBar and add this here:

`currentApp={application.metadata.name} `

Then.....(see comment __HERE__ )